examples/llama/train_llama2.sh

```diff
@@ -0,0 +1,338 @@
+#!/bin/bash
+###############################################################################
+#
+# MIT License
+#
+# Copyright (c) 2024 Advanced Micro Devices, Inc.
+#
```

Comment:
Megatron-LM uses BSD 3-clause license. Please change accordingly.
You can refer to the license information here: https://github.com/ROCm/Megatron-LM/blob/rocm_dev/LICENSE#L5-L27

Reply:
Thanks for updating the license. We do still want to put the AMD copyright statement here:

# Copyright (c) 2024 Advanced Micro Devices, Inc.